glyphs-reader/src/font.rs

```diff
@@ -1502,17 +1502,54 @@ fn default_master_idx(raw_font: &RawFont) -> usize {
                 .enumerate()
```

Comment:
not your code, but instead of enumerate + find, we can use the `position` method..

Reply:
of course, sorry! :) 0b352556